app/templates/components/preprint-form-authors.hbs

```diff
@@ -114,12 +121,12 @@
                                 {{#if contrib.unregisteredContributor}}
                                     {{contrib.unregisteredContributor}}
                                 {{else}}
-                                    <a href={{contrib.users.profileUrl}}> {{contrib.users.fullName}} </a>
+                                    <a href={{contrib.users.links.html}}> {{contrib.users.fullName}} </a>
```

Comment:
Can you clarify why the computed shortcut was replaced with a hardcoded path to the field here?